### PR TITLE
Use keyword paramaters for migration methods for mssql

### PR DESCRIPTION
**Pull request by potiuk**
The new alembic (1.11.0) requires parameters to be keyword parameters in methods used in migrations. Part of the problem has been fixed in #31306 and #31302 but still some migrations that are specifically run in case of mssql need to use migration parameters.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by dstandish — APPROVED** [review]

**Review by eladkal — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -16,6 +16,9 @@
 # specific language governing permissions and limitations
 # under the License.
 """Setup.py for the Airflow project."""
+# This file can be modified if you want to make sure the CI build is using "upgrade to newer dependencies"
+# Which is useful when you want to check if the dependencies are still compatible with the latest versions
+# And they seem to break some unrelated tests in main. You can modify this number = 00001 to trigger it.
```
Comment (pankajkoti) [inline setup.py:21]:
```suggestion
# This file can be modified if you want to make sure the CI build is using "upgrade to newer dependencies",
# which is useful when you want to check if the dependencies are still compatible with the latest versions 
# as they seem to break some unrelated tests in main. You can modify this number = 00001 to trigger it.
```

Comment (pankajkoti) [reply setup.py:21]: also which number can be modified to 00001 we meant here?

Comment (pankajkoti) [reply setup.py:21]: ohh I guess I understood now what was meant here after reading the Slack message. Will it make sense to rephrase this to somewhat like the following?

To make sure the CI build is using "upgrade to newer dependencies", which is useful when you want to check if the dependencies are still compatible with the latest versions as they seem to break some unrelated tests in main, you can modify this file. The modification can be simply modifying this particular comment. e.g. you can modify the following number 00001 to something else to trigger this.

Comment (potiuk) [reply setup.py:21]: Ah. Have not seen the comment> this number (i.e. `00001`) can be modified. But if you think you can describe it better - feel free.

Comment (potiuk) [reply setup.py:21]: Basically any change to `setup.py` will do - even adding a space.

Comment (pankajkoti) [reply setup.py:21]: yeah, GitHub is acting weird, my comments are going late. Earlier I tried to merge a couple of PRs and I felt I did but realised now that they were not merged :(

Comment (pankajkoti) [reply setup.py:21]: Tried to rephrase it here: https://github.com/apache/airflow/pull/31312

Comment (potiuk) [reply setup.py:21]: I think this time I simply did not refresh the PR. I wanted to merge it fast to fix `main` so I missed it :)
